samples/MemoryCacheConcurencySample/project.json

```diff
@@ -2,7 +2,6 @@
     "dependencies": {
         "Microsoft.Extensions.Caching.Memory": "1.0.0-*"
     },
-    "commands": { "MemoryCacheConcurencySample": "MemoryCacheConcurencySample" },
```

Comment:
Leaving the commands doesn't hurt does it? We wont have tooling for a bit and running these things in VS requires the commands. Don't feel strongly, but it might make your lives easier to leave the commands in for the time being.